corehq/ex-submodules/casexml/apps/case/tests/test_ota_restore.py

```diff
@@ -1,32 +0,0 @@
-from django.test import TestCase
-import os
-from django.test.utils import override_settings
-from casexml.apps.case.models import CommCareCase
-from casexml.apps.case.xform import process_cases
-from casexml.apps.case.tests import delete_all_cases
-from couchforms.tests.testutils import post_xform_to_couch
-
-
-@override_settings(CASEXML_FORCE_DOMAIN_CHECK=False)
-class TestOTARestore(TestCase):
```

Comment:
this is funny. some test named something totally wrong that is just a subset of another test. https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/casexml/apps/case/tests/test_multi_case_submits.py#L11